assets/js/laws-regs-parser.js

```diff
@@ -81,14 +83,16 @@ function setUpMobileSearch() {
   const searchBox = searchBoxWrapper.querySelector('.searchbox');
   setUpButtons(searchGo, searchNav, searchBox);
   searchInput.addEventListener('keydown', (e) => {
-    openOverlay(searchBoxWrapper, searchInput);
+    if (e.keyCode != 9) {
```

Comment:
[keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

it really looks like `key == 'Tab'` is the best alternative, as the replacement `code` [has terrible support](https://caniuse.com/?search=keyboardevent.code)